ha/core/config/config_ha.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+"""
+ ****************************************************************************
+ Description:       Provide cental configuration.
+ ****************************************************************************
+"""
+
+from cortx.utils.log import Log
+from cortx.utils.conf_store.conf_store import Conf
+
+from ha import const
+
+# TODO: redefine class in feature as pre design
```

Comment:
"...future as per design"?

Reply:
Currently we need to load log and conf for every new entry point.
To avoid this i created ConfigHA class which acts as central configuration for all process
it help to reduce duplicate code and easy fix.


ConfigHA may not be correct name or may need to modify in future as we planning to implement
config manager which will be responsible for central configuration.
So currently marked as TODO

adding more clear message for next patch